cl/opinion_page/templates/view_opinion.html

```diff
@@ -83,6 +83,28 @@ <h3 class="inline"><span>Your Notes</span></h3>
             </div>
         </div>
 
+        {% if summaries_count > 0  %}
+        <div id="summaries" class="sidebar-section">
+          <h3><span>Summaries ({{ summaries_count|intcomma }})</span></h3>
+          <p class="bottom">Judge-written summaries of this case:</p>
+          <ul>
+            {% for summary in top_summaries %}
+              <li>
+                {{ summary.text }}
+                (from <a href="{{ summary.describing_opinion.cluster.get_absolute_url }}?{{ request.META.QUERY_STRING }}">
+                  {{ summary.describing_opinion.cluster.caption|safe|truncatewords:10|v_wrapper }}</a>)
```

Comment:
These lines give me pause. I wonder two things:

1. Are we likely to get multiple summaries of a case from a single opinion, and if so, would it be wise to group those summaries. So instead of:

    - holding the sky is blue (from Roe v. Wade)
    - finding ham is green (from Roe v. Wade)

    We'd have:

    - From Roe v. Wade:
       -  holding the sky is blue
       - finding ham is green

    A thought.

2. Separate from thought number one, I wonder if the whole summary could/should be the link?

Reply:
And another thought (also applies to the full page template): Should we capitalize the first letter?

Reply:
I agree on capitalizing the first letter, but I'll push back a bit on the other two.

On the first one, I ran some queries on our federal appellate parenthetical corpus. Around 15% of our parentheticals are from opinions that reference the described case in a parenthetical more than once. Less than 1% for more than twice. Scrolling through these instances, my anecdotal impression is that very, very few of these pairs contain parentheticals that would *both* be ranked highly enough to appear in the sidebar. So I'd estimate that the case where more than one parenthetical from the same opinion appears in the sidebar would be quite uncommon.

More broadly, though, (and this applies to the second point as well) I think the main focus of the parentheticals is what they say about the case in front of the user, not its source. The source is definitely relevant context and so we put it afterwards, but it's not the main attraction. For that reason, I don't think it makes sense to give the source hierarchical priority or hyperlink the whole summary to it.

Reply:
Sounds good. I'll look for the capitalization.